config/config.go

```diff
@@ -48,7 +49,7 @@ func (w WatchdogConfig) Process() (string, []string) {
 }
 
 // New create config based upon environmental variables.
-func New(env []string) (WatchdogConfig, error) {
```

Comment:
We were not using the `error` return value so I thought about removing it and cleanup the code a bit.